recipe/kcftracker.patch

```diff
@@ -0,0 +1,11 @@
+--- opencv_contrib/modules/tracking/src/trackerKCF.cpp
++++ opencv_contrib/modules/tracking/src/trackerKCF.cpp
+@@ -666,7 +666,7 @@
+     for(int i=0;i<patch_data.rows;i++){
+       for(int j=0;j<patch_data.cols;j++){
+         pixel=patch_data.at<Vec3b>(i,j);
+-        index=(unsigned)(floor(pixel[2]/8)+32*floor(pixel[1]/8)+32*32*floor(pixel[0]/8));
++        index=(unsigned)(floor((float)pixel[2]/8)+32*floor((float)pixel[1]/8)+32*32*floor((float)pixel[0]/8));
```

Comment:
[Fixed in latest version](https://github.com/Itseez/opencv_contrib/blob/6cd8e9f556c8c55c05178dec05d5277ae00020d9/modules/tracking/src/trackerKCF.cpp#L669)